chart/templates/vmagent/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
```

Comment:
I think, vmagent's configmap need to be updated or removed.
As well as params that are used for its generation need to be updated or removed.

Now, with dynamic config generator, we don't need the static config any more.
This change need to be reflected in README as well, so users will be able to understand ow it works and how to configure it.